tests/src/tools/ReadyToRun.SuperIlc/ParallelRunner.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
```

Comment:
Could you add the copyright header

Reply:
Hmm, for sure, sorry about missing it again :-).